pkg/pillar/cmd/domainmgr/containerd.go

```diff
@@ -197,3 +288,61 @@ func getRktPodManifest(PodManifestFile string) (RktPodManifest, error) {
 	}
 	return manifest, nil
 }
+
+func loadContainer(containerID string) (containerd.Container, error) {
```

Comment:
I suggest you either return a pointer `*containerd.Container` so you can return `nil` if the container doesn't exist, or have a special `error` that you can use to indicate it. There is a difference, which we will use, if the container does not exist (e.g. `ctrRm` doesn't think that is an error, as it is the desired end-state) vs if we had an error.

Reply:
Done.